django/api/scheduled_jobs.py

```diff
@@ -57,25 +48,16 @@ def schedule_expire_expired_applications():
         pass
 
 
-def schedule_upload_verified_applications_last_24hours_to_s3():
```

Comment:
Is this no longer needed?

Reply:
Yeah this is part of the CRA process automation thing, which is on hold; if we are going to do it, it would probably be done differently too